### PR TITLE
Review instances of TimescaleDB

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Reviews each instance of TimescaleDB and updates where appropriate. Plus a few drive-bys, because I can't help myself.

# Links

- Partial: #121 
- Based on: https://github.com/timescale/docs/pull/2355

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!

File: about/contribute-to-docs.md [hunk 1]
```diff
@@ -257,12 +257,12 @@ device.
...
 &#10060; "Perform a bare metal installation."
 
-&#128077; "Install TimescaleDB on bare metal."
+&#128077; "Install Timescale on bare metal."
 
 *Bare-metal*
 : Adjective.
 
-&#10060; "Install TimescaleDB on bare-metal."
+&#10060; "Install Timescale on bare-metal."
```
Comment (gina-alioto) [inline about/contribute-to-docs.md:265]:
```suggestion
&#10060; "Install TimescaleDB on bare-metal."
```
I feel like this is a case where it remains DB.

File: about/contribute-to-docs.md [hunk 2]
```diff
@@ -413,9 +413,9 @@ device.
 *Simply*
 : Don't use.
 
-&#10060; Simply install TimescaleDB.
+&#10060; Simply install Timescale.
```
Comment (gina-alioto) [inline about/contribute-to-docs.md:416]:
```suggestion
&#10060; Simply install TimescaleDB.
```

File: about/contribute-to-docs.md [hunk 3]
```diff
@@ -413,9 +413,9 @@ device.
...
 
-&#128077; Install TimescaleDB.
+&#128077; Install Timescale.
```
Comment (gina-alioto) [inline about/contribute-to-docs.md:418]:
```suggestion
&#128077; Install TimescaleDB.
```

File: about/contribute-to-timescale.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 keywords: [contribute]
 tags: [github]
 ---
 
-# Contributing to TimescaleDB
+# Contributing to Timescale
 
-There are multiple ways to help make TimescaleDB better. All of our documentation
-and source for the PostgreSQL extension are available to use and review via
+There are multiple ways to help make Timescale better. All of the documentation
```
Comment (gina-alioto) [inline about/contribute-to-timescale.md:10]:
```suggestion
There are multiple ways to help make TimescaleDB better. All of the documentation
```

File: about/contribute-to-timescale.md [hunk 5]
```diff
@@ -42,17 +42,17 @@ can ensure that the community is free and confident in its
 ability to use your contributions. You are prompted to sign the
 CLA during the pull request process.
 
-## Contributing to TimescaleDB code
+## Contributing to Timescale code
```
Comment (gina-alioto) [inline about/contribute-to-timescale.md:45]:
```suggestion
## Contributing to TimescaleDB code
```

File: about/contribute-to-timescale.md [hunk 6]
```diff
@@ -42,17 +42,17 @@ can ensure that the community is free and confident in its
...
 
-Timescale appreciates any help the community can provide to make TimescaleDB better!
+Timescale appreciates any help the community can provide to make Timescale better!
```
Comment (gina-alioto) [inline about/contribute-to-timescale.md:47]:
```suggestion
Timescale appreciates any help the community can provide to make TimescaleDB better!
```

File: about/contribute-to-timescale.md [hunk 7]
```diff
@@ -42,17 +42,17 @@ can ensure that the community is free and confident in its
...
 
 There are multiple ways you can help:
 
 *   Open an issue with a bug report, build issue, feature request, suggestion, etc.
 *   Fork this repository and submit a pull request
 
-[Head over to our GitHub repository][github-timescaledb] for TimescaleDB to learn more about how you
-can help and to review our coding style guide!
+[Head over to our GitHub repository][github-timescaledb] for Timescale to learn
```
Comment (gina-alioto) [inline about/contribute-to-timescale.md:54]:
```suggestion
[Head over to our GitHub repository][github-timescaledb] for TimescaleDB to learn
```

**Review by gina-alioto — COMMENTED** [review]
I've made a few comments where I'm questioning whether it should be TimescaleDB over Timescale.  We can discuss on Monday, as I started having doubts the further I got into the review.  When it's on bare metal, it feels like it should be TimescaleDB.  Also, when we are asking for the Community to submit changes, TimescaleDB also feels more appropriate than Timescale.  That said, I'm open to a discussion about it.

File: about/contribute-to-docs.md (hunk 1 above)
Comment (Loquacity) [reply about/contribute-to-docs.md:265]: This is in the Word Usage Guide, it's just an example.

File: about/contribute-to-docs.md (hunk 2 above)
Comment (Loquacity) [reply about/contribute-to-docs.md:416]: Same, this is in the Word Usage Guide.

**Pull request merged**